brave/brave-ui
# Adding tip information to donation overlay

Related to: https://github.com/brave/brave-browser/issues/3023

Storybook: https://brave-ui-kd2ayi16h.now.sh

Pictured changes:

Non-recurring:
<img width="503" alt="screen shot 2019-01-23 at 1 01 04 am" src="https://user-images.githubusercontent.com/8732757/51591665-d2b71c00-1eaa-11e9-8153-702d956733fa.png">

Recurring:
<img width="500" alt="screen shot 2019-01-23 at 1 01 11 am" src="https://user-images.githubusercontent.com/8732757/51591673-dba7ed80-1eaa-11e9-9d4e-7ab63044d1d5.png">


## Test plan
- Go to the Donation Overlay component in the storybook
- Check knobs so that `success=true send=false recurring=yes` to view the updated design for recurring
- Check knobs so that `success=true send=false recurring=no` to view the updated design for non-recurring

## Integration
  
     - When uplift-approved, merge to brave-core-0.VV.x feature branch
     - Create additional brave-core PRs for each feature branch to update commit
